bagua/torch_api/distributed.py

```diff
@@ -60,25 +60,52 @@ def __init__(
             self.parameters_to_ignore = []
 
     def unwarp(self):
+        """
+        Return the unwarmped module.
+        """
+
         return self.module
 
     def forward(self, *inputs, **kwargs):
+        """
+        Execute the forward process and return the output.
+        """
+
         result = self.module(*inputs, **kwargs)
         return result
 
 
 class Reducer(object):
-    """
-    -- New --
-    bucket_initialized is False:
-        1. add_param
-        2. initialize_buckets -> register_models
-        3. mark_bucket_ready
-        4. mark_on_complete
-
-    bucket_initialized is True:
-        1. mark_tensor_ready
-        2. mark_on_complete
+    """ In order to improve communication efficiency, the distributed algorithm divides the parameter \
+        into different blocks, called buckets, and the parameter gradient communicates between different devices in the form of buckets. \
+        This module is the bucket manager，providing bucket operation methods.
+        The process mainly consists following two situations:
+        
+        1. bucket_initialized is False:
+          (1) add_param
+          (2) initialize_buckets -> register_models
+          (3) mark_bucket_ready
+          (4) mark_on_complete
+
+        2. bucket_initialized is True:
+          (1) mark_tensor_ready
+          (2) mark_on_complete 
+
+    Args:
+        module (torch.nn.Module): Module to be parallelized.
+        optimizers (list of optimizer(s)): Optimizer(s) can be single or mutiple. Its mainly used to divided the \
```

Comment:
module is of type `DistributedModule` and optimizers are of type `torch.optim.Optimizer or list of torch.optim.Optimizers`

Reply:
ok